edgelet/contrib/enterprise-linux/aziot-edge.spec

```diff
@@ -62,19 +63,6 @@ make \
 rm -rf $RPM_BUILD_ROOT
 
 %pre
-# Check for container runtime
-if ! /usr/bin/getent group docker >/dev/null; then
-    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
-    echo ""
-    echo " ERROR: No container runtime detected."
-    echo ""
-    echo " Please install a container runtime and run this install again."
-    echo ""
-    echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
-
-    exit 1
-fi
-
 # Create iotedge group
```

Comment:
You need to move all of this to `%post`. What you have right now won't work.

Please test it to make sure it works.